packages/stream-model/src/model.ts

```diff
@@ -158,63 +68,146 @@ async function throwReadOnlyError(): Promise<void> {
   )
 }
 
+/**
+ * Represents the relationship between an instance of this model and the controller account.
+ * 'list' means there can be many instances of this model for a single account. 'link' means
+ * there can be only one instance of this model per account (if a new instance is created it
+ * overrides the old one).
+ */
+export enum ModelAccountRelation {
```

Comment:
In `@glazed/types` we currently have `export declare type ModelAccountRelation = 'list' | 'set' | 'link' | 'none';`

Would it make sense to put this in `common` and use the same types everywhere?